src/main/java/school/lemon/changerequest/java/introduction/hw1/Task10.java

```diff
@@ -3,7 +3,19 @@
 
 public class Task10 {
 
-    public static boolean containDigitTwo(int n) {
-        return false;
-    }
+    public static boolean containDigitTwo(int a) {
+        boolean flag = false;
+        int aLast;
+        if(a == 0)
```

Comment:
No need in this verification and assignment because your flag is already set to false.